packages/lesswrong/components/vulcan-forms/FormWrapper.tsx

```diff
@@ -164,7 +164,7 @@ const FormWrapperEdit = <N extends CollectionNameString>(props: WrappedSmartForm
   const currentUser = useCurrentUser();
   const collection = getCollection(props.collectionName);
   const { queryFragment, mutationFragment } = getFragments("edit", props);
-  const { extraVariables = {}, extraVariablesValues = {}, editFormFetchPolicy } = props;
```

Comment:
What's going on here? Seems risky

Reply:
Optimization to avoid a loading state when a user clicks to edit a JargonTerm - in some cases that can be handled with a different fetch policy, but in this case we're fetching all the terms with a `useMulti` but the smart form uses a `useSingle`, so it won't find the cached record.  I talked to Jim about it.  In this case, it should be safe, since mismatches are pretty unlikely (only really possible if a coauthor edits a jargon term after you've already loaded the post, and then you try to edit it yourself.. but even then, it should still replace the contents after the `useSingle` is done, since it coalesces on `document ?? prefetchedDocument`, which shouldn't take longer than a few hundred ms.)